### PR TITLE
Refine command line support

### LINKED ISSUE #5: Provide a better error message when :color-scheme can not be loaded
If a color scheme cannot be loaded, the error message is very indirect. It should be more helpful.

Examples:

```sh
bash-3.2$ clojure -X com.phronemophobic.membrane.term/run-script :path '"script3.sh"' :width 90 :height 40 :color-scheme :foobar
Execution error (IllegalArgumentException) at com.phronemophobic.membrane.term.color-scheme/load-scheme (color_scheme.clj:65).
Cannot open <:foobar> as an InputStream.
```

```sh
bash-3.2$ clojure -X com.phronemophobic.membrane.term/run-script :path '"script.sh"' :color-scheme '"foobar"'
Namespace could not be loaded: com.phronemophobic.membrane.term
```

### PR DESCRIPTION
**Pull request by lread**
- Employed docopt for basic definition and parsing
  (docopt usage text is the docopt spec)
- The run-script cmd has been renamed to screenshot
  - its path option has been renamed to play
- Added specific validations for numeric, path and image out
  cmd line args. Closes #5
- Updated README
  - Added Installation section
  - We describe the new -M --option convention only.
    The -X syntax is still available but has no new validation and is
    arguably less usable.
  - To eat our own dog food, added script/regen-screenshots.sh,
    which generates new images for screenshot usages fleshed out
    in README.
- Root dir was getting a bit cluttered, so moved images and new
  doc examples under doc.
- Added doc/dev.md to jot a note about script/regen-screenshots.sh

### CODE REVIEW

**Comment by lread** [comment]
Lemme know watcha think.
Even though I reviewed several times, I'm sure there must be some typos for you to find!

**Comment by lread** [comment]
New error reporting for #5 

```
> clj -M:membrane.term run-term --color-scheme :foobar
*
* Error: --color-scheme: unable to open path, :foobar (No such file or directory)
*

membrane.term

Usage:
  membrane.term run-term   [--width=<cols>] [--height=<rows>] [--color-scheme=<path>]
  membrane.term screenshot --play=<path> [--width=<cols>] [--height=<rows>] [--color-scheme=<path>] [--out=<file>] [--line-delay=<ms>] [--final-delay=<ms>]
  membrane.term --help

Common Options:
  -w, --width=<cols>         Width in characters [default: 90]
  -h, --height=<rows>        Height in characters [default: 30]
  -s, --color-scheme=<path>  Local path or url to iTerm .itermcolors scheme file, uses internal scheme by default.

Screenshot Options:
  -p, --play=<path>          Path to script to play in terminal
  -o, --out=<file>           Filename of the image to generate [default: terminal.png]
  -l, --line-delay=<ms>      Delay in ms to wait after each line is sent to the terminal [default: 1000]
  -f, --final-delay=<ms>     Delay in ms to wait before writing the view of the terminal [default: 10000]

Replace membrane.term with your appropriate Clojure tools CLI launch sequence. For example:
|
| clojure -M:membrane.term run-term -w 133 -h 60
|
```

File: README.md [hunk 1]
```diff
@@ -10,64 +10,140 @@ Some reasons to use **membrane.term**:
...
+## Installation
+
+Membrane.term works on macOS and Linux operating systems.
+
+To invoke via `clj -M:membrane.term` add the following alias to your `deps.edn`:
+```Clojure
+{:aliases
+ {:membrane.term
+  {:replace-deps
+   {phronmophobic/membrane.term {:git/url "https://github.com/phronmophobic/membrane.term.git"
```
Comment (phronmophobic) [inline README.md:24]: should be com.phronemophobic/membrane.term. I plan on putting membrane.term on clojars under the same name

**Comment by phronmophobic** [comment]
So good! Thank you!

**Comment by lread** [comment]
Cool, glad it works for you too!

**Pull request merged**